utils/container.hpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018 Bluzelle
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License, version 3,
+// as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+#include <include/bluzelle.hpp>
+#include <bootstrap/bootstrap_peers_base.hpp>
+#include <random>
+
+namespace
+{
+    std::mt19937 gen(std::time(0)); //Standard mersenne_twister_engine seeded with rd()
```

Comment:
anywhere we include this it will be initialized? Not sure the value of the container.hpp.

Reply:
Yeah, that's a bad... I'll create a container.cpp, and break it up, for now. 

You know, what? I'll blow away the container.hpp and just move the function into the module where it's used..